tdp/core/runner/operation_runner.py

```diff
@@ -120,7 +120,10 @@ def from_dag(
         else:
             str_filter = None
             filter_type = None
-
+        if len(operation_names) == 0:
+            raise ValueError(
```

Comment:
I think we should not raise this generic exception, this code is part of a lib and the exception should be specific like `EmptyListOfOperations` but in case we do not raise exception we must decide if returning an `OperationPlan` without operation to run make sense. Other solution is to return `None` instead of an `OperationPlan`.

Reply:
It's done